### DISCUSSION TITLE
Can we migrate to KRaft using JBOD with multiple disks?

### DISCUSSION THREAD

**Discussion by bavaria-blue** [strimzi/strimzi-kafka-operator#10878] [Q&A] [answered]
I'm unclear on whether JBOD with multiple disks is still a problem when migrating to KRaft.

The [latest Strimzi documentation](https://strimzi.io/docs/operators/latest/deploying#proc-deploy-migrate-kraft-str) says:

> Verify that the ZooKeeper-based deployment is operating without the following, as they are not supported in KRaft mode:
> - JBOD storage. While the jbod storage type can be used, the JBOD array must contain only one disk.

But other documentation I've seen seems to imply that with Kafka 3.8.0 there should be no problems with migration to KRaft.

We have storage type JBOD with multiple disks and are using Strimzi 0.43 and Kafka 3.8.0. Can we migrate to KRaft without worry or is there something we should wait for?

Thanks in advance for your response.

**Comment by scholzj** [comment]
I think this should be supported to the extend Apache Kafka supports it. Or @ppatierno?

**Reply by ppatierno** [reply, marked as answer]
Jakub is right. If you are using a Kafka version which supports multiple JBOD disks, you can run the migration.
The Strimzi migration code itself doesn't do anything special around disks. It just orchestrate the migration. It's Kafka dealing with disks during the running migration. I think that sentence in the Strimzi doc is mostly referring to the minimum Kafka version for migration 3.7.0, which has support for only one JBOD disk.

**Reply by bavaria-blue** [reply]
OK, thanks for the quick response.

**Reply by scholzj** [reply]
I think this can definitely be confusing. I opened https://github.com/strimzi/strimzi-kafka-operator/pull/10879 to remove it.
